pages/core-concepts/writing-and-organizing-tests.md

```diff
@@ -0,0 +1,673 @@
+# Escrevendo e Organizando Testes
+
+> ## __O que você aprenderá__
+>
+>- Como organizar seus arquivos de teste e suporte.
+>
+>- Quais idiomas são suportados em seus arquivos de teste.
+>
+>- Como o Cypress lida com testes de unidade versus testes de integração.
+>
+>- Como agrupar seus testes.
+
+> __Melhores Práticas__
+>
+>Recentemente demos uma palestra de “Melhores práticas” na conferência AssertJS
+>(Fevereiro de 2018). Este vídeo demonstra
+>como dividir seu aplicativo e organizar seus testes.
+>
+><https://www.youtube.com/watch?v=5XQOK0v_YRE>
+
+## Estrutura da Pasta
+
+Depois de adicionar um novo projeto, o Cypress criará automaticamente uma estrutura de pastas sugerida. Por padrão, ele criará:
+
+```markdown
+/cypress
+  /fixtures
+    - example.json
+
+  /integration
+    /examples
+      - actions.spec.js
+      - aliasing.spec.js
+      - assertions.spec.js
+      - connectors.spec.js
+      - cookies.spec.js
+      - cypress_api.spec.js
+      - files.spec.js
+      - local_storage.spec.js
+      - location.spec.js
+      - misc.spec.js
+      - navigation.spec.js
+      - network_requests.spec.js
+      - querying.spec.js
+      - spies_stubs_clocks.spec.js
+      - traversal.spec.js
+      - utilities.spec.js
+      - viewport.spec.js
+      - waiting.spec.js
+      - window.spec.js
+
+  /plugins
+    - index.js
+
+  /support
+    - commands.js
+    - index.js
+  ```
+
+### Configurando a Estrutura da Pasta
+
+Enquanto o Cypress permite que você configure onde seus testes, fixtures
+e arquivos de suporte estão localizados, se você estiver iniciando
+seu primeiro projeto, recomendamos que você use a estrutura acima.
+
+Você pode modificar a configuração da pasta em seu arquivo de configuração.
+Veja a [configuração](https://docs.cypress.io/guides/references/configuration.html#Folders-Files) para mais detalhes.
+
+>__Quais arquivos devo adicionar ao meu arquivo '.gitignore'?__
+>
+>O Cypress criará um [`screenshotsFolder`](https://docs.cypress.io/guides/references/configuration.html#Screenshots) e um
+>[`videosFolder`](https://docs.cypress.io/guides/references/configuration.html#Videos) para armazenar as capturas de tela
+>e vídeos feitos durante o teste de seu aplicativo. Muitos usuários optarão por adicionar essas pastas a seus arquivos `.gitignore`.
+>Além disso, se você estiver armazenando variáveis ​​de ambiente confidenciais em seu arquivo de configuração
+>(`cypress.json` por padrão) ou [`cypress.env.json`](https://docs.cypress.io/guides/guides/environment-variables.html#Option-2-cypress-env-json),
+>elas também devem ser ignoradas ao fazer check-in no controle de origem.
```

Comment:
A que se refere controle de origem? Seria controle de versão de código?

Reply:
Opa @lucastfa-toptal. A doc fala em "these should also be ignored when you check into source control." Seria algo como "elas também devem ser ignoradas da mesma forma."? Diz respeito à inclusão de mais informações ao .gitignore por se tratar de informação sensível.

Reply:
Oi @rafaabc! Entendi, nesse caso gosto da sua sugestão. Ou então pode traduzir source control pra sistema de versionamento de código, ou até mesmo Git já que o texto menciona .gitignore explicitamente. "elas também devem ser ignoradas ao fazer check-in no Git".